osbs/cli/main.py

```diff
@@ -406,14 +401,14 @@ def cli():
                         help="Export build metadata as JSON file")
     args = parser.parse_args()
 
-    if getattr(args, 'func', None) is cmd_build_source_container:
+    if getattr(args, 'func') is cmd_build_source_container:
```

Comment:
Later in `main()` there is logic about printing the help text if the `func` attribute is not set. If that's not relevant anymore, should it also be cleaned up? Does the parser guarantee that func is set if `parse_args()` succeeds?

Reply:
hmm, we do need to show help. fixed this to default to `None` instead of failing